nevergrad/functions/base.py

```diff
@@ -204,6 +204,36 @@ def update_leaderboard(identifier: str, loss: float, array: np.ndarray, verbose:
         pass  # better avoir bugs for this
 
 
+class ArrayExperimentFunction(ExperimentFunction):
+    """Adds a "symmetry" parameter, which allows the creation of many symmetries of a given function."""
+
+    def symmetrized_function(self: AEF, x: tp.Any):
+        assert isinstance(x, np.ndarray), "symmetry != 0 works only when the input is an array."
+        assert len(x.shape) == 1, "only one-dimensional arrays for now."
+        y = x
+        symmetry: int = self._symmetry  # type: ignore
+        for i in range(len(y)):
+            if symmetry % 2 == 1:
+                y[i] = -x[i]  # We should rather symmetrize w.r.t the center of Parameter. TODO
+            symmetry = symmetry // 2
+        return self._inner_function(y)  # type: ignore
+
+    def __init__(self: AEF, function: tp.Callable[..., tp.Loss], parametrization: p.Parameter, symmetry: int = 0) -> None:
+        """ Same parameters as ExperimentFunction, plus "symmetry".
+        symmetry: an int, 0 by default.
+        if not zero, a symmetrization is applied to the input; each of the 2^d possible values
+        for symmetry % 2^d gives one different function.
+        Makes sense if and only if (1) the input is a single ndarray (2) the domains are symmetric."""
```

Comment:
Please move the init as the first method, and all class related docstring should be in the init, hence you should remove 
"""Adds a "symmetry" parameter, which allows the creation of many symmetries of a given function."""
from  above and add it here

Reply:
Done.